packages/create-cloudflare/src/common.ts

```diff
@@ -325,8 +326,8 @@ export const gitCommit = async (ctx: PagesGeneratorContext) => {
 	//       we unconditionally run this command here
 	const commitMessage = await createCommitMessage(ctx);
 
-	if (!(await isGitInstalled()) || !(await isInsideGitRepo(ctx.project.path)))
-		return;
+	// we actually commit only if c3 was the one initializing the git repository
+	if (!ctx.c3InitializedGitRepo) return;
```

Comment:
instead of doing this here we could prompt the user asking if they'd want us to commit for them, I think I'd prefer not to do that because:
- in this was we keep things simple and clean, no potential edge cases or risk to committing things wrongly (e.g. monorepos with dirty directories)
- maintain a very clear/basic behavior (clear/basic both for us and users): when we initialize the git repo we also generate the first commit, there are no other possible cases/alternative behaviors
- this seems to be the general behavior of such type of CLI tools
- asking the user at this point feels a bit pointless, as they can just committing themselves (the only benefit we could bring would be that we can create a commit with all the C3 details, but I am not sure if people really care about it)


But let me know what you think, if someone strongly prefers prompting the user and potentially committing the changes I am also ok with doing that 🙂